yb-voyager/src/srcdb/postgres.go

```diff
@@ -180,3 +184,41 @@ func (pg *PostgreSQL) ExportDataPostProcessing(exportDir string, tablesProgressM
 	}
 	dfd.Save()
 }
+
+// Given a PG command name ("pg_dump", "pg_restore"), find absolute path of
+// the executable file having version >= `PG_COMMAND_VERSION`.
+func GetAbsPathOfPGCommand(cmd string) (string, error) {
+	paths, err := findAllExecutablesInPath(cmd)
+	if err != nil {
+		err = fmt.Errorf("error in finding executables: %v", err)
+		return "", err
+	}
+	if len(paths) == 0 {
+		err = fmt.Errorf("the command %v is not installed", cmd)
+		return "", err
+	}
+
+	for _, path := range paths {
+		cmd := exec.Command(path, "--version")
+		stdout, err := cmd.Output()
+		if err != nil {
+			err = fmt.Errorf("error in finding version of %v from path %v: %v", cmd, path, err)
```

Comment:
`%w` is the format specifier used to wrap `error` objects.

Reply:
Oh right